config.go

```diff
@@ -40,6 +40,9 @@ type SpanNameFormatter func(ctx context.Context, method Method, query string) st
 // AttributesGetter provides additional attributes on spans creation.
 type AttributesGetter func(ctx context.Context, method Method, query string, args []driver.NamedValue) []attribute.KeyValue
 
+// InstrumentAttributesGetter provides additional attributes while recording metrics.
```

Comment:
```suggestion
// InstrumentAttributesGetter provides additional attributes while recording metrics to instruments.
```

nit, as `instrument` is the concrete type for capturing metrics in OpenTelemetry. https://opentelemetry.io/docs/concepts/signals/metrics/#metric-instruments

Having mentioned this, the implementation is straightforward under the hood.